content/en/blog/news/first-post/index.md

```diff
@@ -1,17 +1,129 @@
 ---
-date: 2024-02-28
+date: 2024-03-13
 title: Introducing SpinKube
 linkTitle: Introducing SpinKube
 description: >
-  A new open source project that streamlines the experience of developing, deploying, and operating Wasm workloads on Kubernetes.
-author: Fermyon Technologies ([@fermyontech](https://twitter.com/fermyontech))
+  Bringing the next wave of hyper-efficient serverless to Kubernetes.
+author: The SpinKube Team ([@SpinKube](https://mastodon.social/@SpinKube))
 resources:
   - src: "**.{png,jpg}"
     title: "Image #:counter"
 ---
+Today we're introducing SpinKube - an open source platform for efficiently 
+running Spin-based WebAssembly applications on Kubernetes. Built with love by 
+folks from Microsoft, SUSE, LiquidReply, and Fermyon.
 
-# Introducing SpinKube
+SpinKube combines the application lifecycle management of the [Spin Operator][spin-operator], the efficiency of the [containerd-shim-spin][containerd-shim-spin], and the node lifecycle management of the forthcoming [runtime-class-manager][runtime-class-manager] (formerly [KWasm][kwasm]) to provide an excellent developer and operator experience alongside excellent density and scaling characteristics regardless of processor architecture or deployment environment.
 
+>
+> TL;DR? Check out the [quickstart][quickstart] to learn how to deploy
```

Comment:
```suggestion
> TL;DR? Check out the [quickstart](https://www.spinkube.dev/docs/spin-operator/quickstart/) to learn how to deploy
```

Reply:
This is a reference-style link and is considered valid markdown.